### ISSUE TITLE
'Tree Hybrid' support ignores 'Top Z Distance' for some geometries, making support removal difficult

### ISSUE THREAD

**Issue by AJZein** [bambulab/BambuStudio#4194]
### Bambu Studio Version

1.9.1.66

### Where is the application from?

Bambu Lab github releases

### OS version

Windows 10

### Printer

Bambu Lab P1P

### How to reproduce

Download and slice the included .3mf file. Seems to only occur on curved overhangs.

### Actual results

The support interface has nearly no gap from the model making supports difficult to remove. Setting 'Top Z Distance' does not affect the gap at all (set here to 0.5mm)
![image](https://github.com/bambulab/BambuStudio/assets/45639168/87ad8e04-fd33-4896-b3c0-eec2fbbab3f3)

### Expected results

The support interface should have the set gap from the model to facilitate easy removal. 'Tree Organic' seems to better handle this and respond to changes in 'Top Z Distance' as seen below
![image](https://github.com/bambulab/BambuStudio/assets/45639168/dbe8b457-c449-4164-aa92-c0bab3cc43b3)

### Project file & Debug log uploads

[tree_hybrid_test.zip](https://github.com/user-attachments/files/15524878/tree_hybrid_test.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

**Comment by AJZein** [comment]
After some more testing 'Tree Strong' and 'Tree Slim' are also affected by this issue

I've added a closer look below. Using Top Z Distance of 0.6mm and a layer height of 0.2mm, I would expect 3 layers of separation between interface and model. However there seems to be none in this example. For reference, normal grid supports show the expected 3 layer gap.

Layer 82, no interface on the left
![image](https://github.com/bambulab/BambuStudio/assets/45639168/49b87324-da0f-4770-8a2a-d5750b4b5d9d)

Layer 83, interface constructed on the left
![image](https://github.com/bambulab/BambuStudio/assets/45639168/9d3d2007-9b3f-4c7e-910a-bce08a56e26f)

Layer 84, overhang already on interface. Next interface layer created
![image](https://github.com/bambulab/BambuStudio/assets/45639168/5b61288f-c9c9-4d90-ae5d-ce4cdcef9041)

And the same repeats for the next layers
![image](https://github.com/bambulab/BambuStudio/assets/45639168/7d6fac4c-3b71-4103-9f16-a8ad85f6ba69)
![image](https://github.com/bambulab/BambuStudio/assets/45639168/10d17d3b-de34-4956-b90e-7903eeec8c40)

**Comment by AJZein** [comment]
Updated to version 1.9.2.57, issue seems almost resolved with 'Independent support layer height' enabled. However when I disabled this to count the layers accurately the issue re-appeared 

Enabled
![image](https://github.com/bambulab/BambuStudio/assets/45639168/fd4f8fdf-901d-4928-94eb-89d981016141)

Disabled
![image](https://github.com/bambulab/BambuStudio/assets/45639168/4d6185ea-4b65-4d81-a4c7-bdf36a72c877)

**Comment by tangketan** [comment]
Yes, synced support layer height still has issues... Thanks for reporting. I'll fix it soon.

**Comment by ClawzSkunk** [comment]
> Yes, synced support layer height still has issues... Thanks for reporting. I'll fix it soon.

Can you advise when? I've had issues with supports for quite some time now

**Comment by fahaaaxiki** [comment]
@ClawzSkunk Next Bambu Studio version will fix this problem.

**Comment by ClawzSkunk** [comment]
Thankyou for the update. Is there an expected date for the next update?
